pkg/daemon/metrics.go

```diff
@@ -126,6 +126,15 @@ var (
 			Help:      "0 = FREERUN, 1 = LOCKED, 2 = HOLDOVER",
 		}, []string{"process", "node", "iface"})
 
+	// ClockClassMetrics metrics to show current clock class
+	ClockClassMetrics = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: PTPNamespace,
+			Subsystem: PTPSubsystem,
+			Name:      "clock_class",
+			Help:      "6 = Locked,7 = PRC unlocked but still in spec,52, 187 =  PRC unlocked and  out of spec,248 = Default,255 =  Slave Only Clock",
```

Comment:
"6 = Locked, 7 = PRC unlocked in-spec, 52/187 =  PRC unlocked out-of-spec, 248 = Default, 255 = Slave Only Clock"